src/MizarData/include/MizarData/MizarPairedData.h

```diff
@@ -159,11 +161,14 @@ class MizarPairedDataTmpl {
     return count;
   }
 
+  [[nodiscard]] uint64_t ToUint64TTimestampNs(RelativeTimeNs relative_time) const {
```

Comment:
I would call that `ToUint64TAbsoluteTimestampNs` or something like this, to make it more clear that this also adds `data_->GetCaptureStartTimestampNs()`. (Just from reading the current name, I expected a simple conversion from the RelativeTimeNs type to `uint64_t`.

Reply:
Let's call it `ToAbsoluteTimestamp`